server/models/product.py

```diff
@@ -5,17 +5,22 @@ class Product(db.Model):
 
     __tablename__ = 'product'
 
-    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    id = db.Column(db.String, primary_key=True)
     name = db.Column(db.String, unique=True, nullable=False)
+    currency = db.Column(db.String, nullable=False)
     old_price = db.Column(db.Integer, nullable=False)
     price = db.Column(db.Integer, nullable=False)
+    availability = db.Column(db.Boolean, nullable=False)
     url = db.Column(db.String, nullable=False)
     img_url = db.Column(db.String, nullable=False)
 
-    def __init__(self, name, old_price, price, url, img_url):
+    def __init__(self, id, name, currency, old_price, price, availability, url, img_url):
+        self.id = id
         self.name = name
+        self.currency = currency
```

Comment:
I realized, you have a currency column for both product and prices table, I don't think this would be necessary for the prices table.